packages/app/pages/login.tsx

```diff
@@ -13,86 +12,12 @@ const Container = styled.div`
   width: 300px;
   padding-top: 100px;
 `;
-class option {
-  value: number;
-  label: string;
-}
+
 export default function Login(): ReactElement {
-  const [isModalOpen, setIsModalOpen] = useState(false);
   const [pass, setPass] = useState("");
   const [uname, setUname] = useState("");
-  const [cOptions, setCOptions] = useState<option[]>(null);
```

Comment:
Refactoring

Reply:
this is good, I like the refactor of the login part